packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-env browser */
+
+/* global Webex */
+
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable no-global-assign */
+/* eslint-disable no-multi-assign */
+
+// Globals
+let webex;
+
+// Collapsible Setup
```

Comment:
I like longer comment delineators, I find it makes it easier to glance at sections
```suggestion
// Collapsible Setup --------------------------------------------------
```

Reply:
Implemented.